Docs/source/running_cpp/parameters.rst

```diff
@@ -279,7 +279,7 @@ Particle initialization
       symmetrize the beam in the x and y directions).
 
     * ``external_file``: inject macroparticles with properties (charge, mass, position, and momentum) according to data in external file.
-      It requires the additional argument ``<species_name>.injection_file``, which is the string corresponding to the OpenPMD file name.
+      It requires the additional arguments ``<species_name>.injection_file`` and ``<species_name>.q_tot``, which are the string corresponding to the OpenPMD file name and the beam charge.
```

Comment:
Do we want to make `q_tot`, which re-scales the beam charge, mandatory?
Because given the input's scaling, we are able to know the total charge of the injected beam (and re-scaling it might be optional).

```suggestion
      It requires the additional arguments ``<species_name>.injection_file`` and ``<species_name>.q_tot``, which are the string corresponding to the openPMD file name and the beam charge.
```

Reply:
Thanks @ax3l !
And sorry, I had merged the changes of the other PR into this one, so I don't know why the capital letter escaped me...
I don't understand your point.
Right now I want to have information on for example 4 physical particles and then use the `q_tot` parameter to explain that they represent 1nC.
What would you suggest to do differently?

Reply:
Yes, I was wondering since openPMD particles have a weighting and a charge: the input file itself can already represent a beam of a certain charge. Specifying `q_tot` effectively overwrites that information that is contained in the input file.

Due to that, I was wondering if the re-scaling that `q_tot` performs to the weighting of the particles in the input file should be optional. But we can implement such features also later on, don't worry.

Reply:
Sorry @ax3l , probably I am not understanding your point well.
Each physical particle is stored in the openPM file with its charge and mass, yes.
But those physical particles are a sample of a real beam (for example 100 electrons out of a 1nC beam).
Then the user specifies how many particles are in the whole realistic beam, via q_tot.
And WarpX uses that information to launch a number of macroparticles that is the same as the number of physical particles in openPMD has, but with a weight so that they  represent the whole realistic beam (instead of just the sample).
For now the users can already match `q_tot` to the total of physical particles charge in the file, then it is as if they don't use the parameter.
That is why I don't think that `q_tot` needs to be optional.

But just so I understand better your point, why do you want `q_tot` to be optional?
Are you thinking of having an openPMD file with position/momentum information on 10^9 physical electrons?
And implementing all 10^9 into WarpX?

ps: it's a very minor change so I can totally do it in the next PR, of course :)

Reply:
As we discussed offline, in the Geant4 code that I am using to produce the openPMD file with species information, all particles are physical. And I wish to make a scan of simulations, based on the configuration described in the file, where I change the "physical real beam charge" they represent, to determine the best scenario.

The optional `q_tot` variable is being implemented as such in part 3 of this list of PRs.